src/sentry/api/endpoints/organization_stats.py

```diff
@@ -91,8 +91,8 @@ def get(self, request: Request, organization) -> Response:
         data = tsdb.get_range(
             model=stat_model,
             keys=keys,
-            **self._parse_args(request, **query_kwargs),
             tenant_ids={"organization_id": organization.id},
+            **self._parse_args(request, **query_kwargs),
```

Comment:
any particular reason you switched this ordering?

Reply:
Ah it wasn't working and I thought it could be something to do with that `parse_args` function, I'll undo this cause it wasn't the culprit.